client/catalog/src/app/datatypes/DatatypeList.svelte

```diff
@@ -0,0 +1,68 @@
+<script>
+import client from '../../data/client.js';
+import {Runtime} from '../../data/enums.js';
+import {cap, restricts} from '../util.js';
+
+let load = client.datatype.findMany({	include: {
+		validator: true,
+	}
+});
+
+import { breadcrumbStore } from '../stores.js';
+breadcrumbStore.set(
+	[{
+		href: "/datatypes",
+		text: "Datatypes",
+	}]
+);
+
+</script>
+
+<style>
+	.box {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+	}
+	.stuff {
+
+	}
+	a.object-box {
+		display: flex;
+		flex-direction: column;
+		color: inherit;
+		width: 150px;
+		padding: 1em 1.5em;
+	}
+	a.object-box:hover {
+		background: var(--background-highlight);
+	}
+
+	.spacer {
+		width: 0;
+	}
+	.obj-title{
+		font-weight: 600;
+	}
+
+</style>
+
+<div class="box">
+	{#await load}
+		&nbsp;
```

Comment:
oh, i noticed just now that you can {#await load then datatypes} and skip the &nbsp;